addon/components/frost-url-input.js

```diff
@@ -106,7 +106,7 @@ export default Component.extend({
         return
       }
 
-      // this.send('clear')
+      this.send('clear')
```

Comment:
Isn't this going to clear the input as well? I think we don't want that but we want it to clear the status from previous test runs.